### ISSUE TITLE
fastlane sigh resigning with ad-hoc licensing not working

### ISSUE THREAD

**Issue by bernardosamorim** [fastlane/fastlane#9955]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
I am not being able to deploy my app to a registered device on my team account once I resign it with an ad-hoc license, in spite of the resign command finishing successfully - Uploading the app to the device through the iTunes seems to work at first, but once the installation is complete, its icon automatically disappears from the device.

##### Complete output when running fastlane, including the stack trace and command used
`>>fastlane sigh resign ios-\@bsamorim-drop-count-23b3da38-78d1-11e7-b725-000c293c825d-archive.ipa --signing_identity "iPhone distribution: My-Company" -p "/Users/bsamorim/Library/MobileDevice/c2bc896a-1912-4954-8db6-eee58d84d098.mobileprovision" `
`/Users/bsamorim/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.50.1/sigh/lib/assets/resign.sh ios-@bsamorim-drop-count-23b3da38-78d1-11e7-b725-000c293c825d-archive.ipa 3CA286A8DAE229FBCAC2189E26CE65C4A97F3974 -p /Users/bsamorim/Library/MobileDevice/c2bc896a-1912-4954-8db6-eee58d84d098.mobileprovision          ios-@bsamorim-drop-count-23b3da38-78d1-11e7-b725-000c293c825d-archive.ipa`
`security: SecPolicySetValue: One or more parameters passed to a function were not valid.`
`Print: Entry, ":CFBundleDisplayName", Does Not Exist`
`security: SecPolicySetValue: One or more parameters passed to a function were not valid.`
`security: SecPolicySetValue: One or more parameters passed to a function were not valid.`
`_floatsignTemp/Payload/Exponent.app: replacing existing signature`
`rm: _floatsignTemp/old-embedded-profile.plist: No such file or directory`

`[04:28:05]: Successfully signed ios-@bsamorim-drop-count-23b3da38-78d1-11e7-b725-000c293c825d-archive.ipa!`
### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.5                                     |
| Ruby                        | 2.2.4                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.11.0 (Apple Git-81)           |
| Installation Source         | ~/.fastlane/bin/bundle/bin/fastlane         |
| Host                        | Mac OS X 10.12.5 (16F73)                    |
| Ruby Lib Dir                | ~/.fastlane/bin/bundle/lib                  |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | true                                        |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.3                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.50.1  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.1        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.4        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.6        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.55.0       |
| faraday                   | 0.12.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| uber                      | 0.0.15       |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.2        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.12.0       |
| json                      | 1.8.1        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.0        |
| bundler                   | 1.14.6       |
| faraday_middleware        | 0.11.0.1     |
</details>

*generated on:* **2017-08-04**
</details>

**Comment by bernardosamorim** [comment]
Yes, I've looked into the Code Signing Troubleshooting Guide - all certificates for my adhoc provisioning profile were made successfully with the help of `fastlane match`

**Comment by ohayon** [comment]
Hey @bernardosamorim - Would you mind sharing the complete output from your command line so we can have as much context about the issue as possible? Thanks!! 🚀

**Comment by bernardosamorim** [comment]
That is the complete output from the `fastlane sigh resign` command. Is there some other command whose output you'd like me to share?
